.github/release.yml

```diff
@@ -7,6 +7,9 @@ changelog:
     - title: 🐛 Bug Fixes
       labels:
         - bug
+    - title: 📚 Documentation
+      labels:
+        - docs
```

Comment:
With the changes of #25 we should highlight changes to documentation between releases IMO!

Even if these changes are sometimes "unversioned" it can still be so interesting to follow along with these updates 👾

Reply:
++++ totally agree! Thanks for adding this!